src/navmeas.rs

```diff
@@ -143,38 +143,6 @@ impl Default for NavigationMeasurement {
     }
 }
 
-#[cfg(feature = "sbp-conversions")]
-impl std::convert::TryFrom<sbp::messages::observation::PackedObsContent> for NavigationMeasurement {
-    type Error = crate::signal::InvalidGnssSignal;
-
-    fn try_from(
-        observation: sbp::messages::observation::PackedObsContent,
-    ) -> Result<NavigationMeasurement, crate::signal::InvalidGnssSignal> {
-        use std::convert::TryInto;
-
-        let mut measurement = NavigationMeasurement::new();
-
-        measurement.set_lock_time(decode_lock_time(observation.lock));
-        measurement.set_sid(observation.sid.try_into()?);
-        // A CN0 of 0 is considered invalid
-        if observation.cn0 != 0 {
-            measurement.set_cn0(observation.cn0 as f64 / 4.);
-        }
-        if observation.flags & 0x01 != 0 {
-            measurement.set_pseudorange(observation.P as f64 / 5e1);
-        }
-        if observation.flags & 0x08 != 0 {
-            measurement
-                .set_measured_doppler(observation.D.i as f64 + (observation.D.f as f64) / 256.);
-        }
-        if observation.flags & 0x80 != 0 {
-            measurement.0.flags |= NAV_MEAS_FLAG_RAIM_EXCLUSION;
-        }
```

Comment:
I was able to copy/paste pretty much everything expect this bit here `NAV_MEAS_FLAG_RAIM_EXCLUSION` and `measurement.0.flags` are private. Not sure the best way to handle this, maybe add get/setters for the flags?

Reply:
I think it's fair to just make those public, @jbangelo thoughts?

Reply:
Probably the easiest way but seems like there might be potential pitfalls in exposing the `c_bindings` struct directly, but I don't actually know

Reply:
Oh, right, then an accessor is probably better

Reply:
Yeah, a getter/setter pair is preferable since changing that value would involve accessing one of the C struct fields. I wasn't very disciplined in added getter/setters for all the fields so feel free to add any you end up needing. I mostly added them for the fields I needed to tweak while testing the single epoch solver.